### ISSUE TITLE
ComboBox does not clear value with X

### ISSUE THREAD

**Issue by ipeshev** [telerik/kendo-angular#1931]
## I'm submitting a...

* Regression report 

## Current behavior

Clearing value with x icons does not work when [allowCustom]="true"

## Expected behavior
Clearing value should work in any case

## Minimal reproduction of the problem with instructions
1. Open this [demo](https://www.telerik.com/kendo-angular-ui/components/dropdowns/combobox/custom-values/).
2. Try to select and clear value
## Environment

Package versions:

```
broken in v3.2.1-dev.201810301023 
working in v3.2.1-dev.201810301005
```

Browser:

- Chrome (desktop) version XX
- Chrome (Android) version XX
- Chrome (iOS) version XX
- Firefox version XX
- Safari (desktop) version XX
- Safari (iOS) version XX
- IE version XX
- Edge version XX

System:
- TypeScript version: XX 
- Node version: XX  
- Platform:

**Comment by dimitar-pechev** [comment]
Fixed in the [dev build](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) of the `@progress/kendo-angular-dropdowns` package **v3.4.1-dev.201811231239**.

**Comment by dimitar-pechev** [comment]
If it's urgent, you can get the **dev** package from **npm** as described in the shared link from my previous comment. 

The fix will probably be included in the official release next week.

**Comment by dimitar-pechev** [comment]
The fix is shipped in **v3.4.1** of the `@progress/kendo-angular-dropdowns` package.
